### ISSUE TITLE
deprecate package in favor of ember-auto-import and setting up chai directly?

### ISSUE THREAD

**Issue by fivetanley** [ember-cli/ember-cli-chai#358]
Ember CLI Chai has served us well, but its functionality could be replicated by using ember-auto-import. This addon is using several deprecated babel plugins/npm packages that would just be more maintenance over time, and nice to remove from the tree of packages. I'd be happy to submit a PR to the README with the replacement instructions, which are:

Add Ember Auto Import:

`ember install ember-auto-import`

Remove ember-cli-chai from package.json

import `chai` and plugins into your `test-helper.js`:

```javascript
import chai from 'chai';
// any other chai plugins from your package.json
import chaiJquery from 'chai-jquery';

chai.use(chaiJquery);
```

**Comment by Turbo87** [comment]
sounds reasonable to me.

@rwjblue any objections?

**Comment by rwjblue** [comment]
None. I plan to do something similar with ember-qunit: moving `qunit` and `@ember/test-helpers` into the application blueprint, setting them up as `peerDependencies`, and allowing `ember-auto-import` to discover and bundle the `qunit` dependency.

**Comment by Turbo87** [comment]
resolved by #363
